README.md

```diff
@@ -1,10 +1,9 @@
 # Evaluate
 
-This repository is NOT A SECURE IMPLEMENTATION!
+INSECURE three-argument evaluate function.
 
-You should be using [SES](https://github.com/Agoric/SES) or
-[realms-shim](https://github.com/Agoric/realms-shim) to call `evaluate`
-with secure control of global variables.
+You should be using [Secure ECMAScript](https://github.com/Agoric/SES) or Realms
+to call `evaluate` with security in mind.
```

Comment:
maybe add a note to explain what we mean by insecure here. "The evaluated code will have full access to the globals and something something scope of the caller, which is far more authority than you really want to give that code".

Reply:
Thanks, this encouraged me to prevent one form of leakage.